apps/web/src/hooks/useIsAuxiliaryFundsEnabled.ts

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from 'react';
+import { useAccount } from 'wagmi';
+import { useCapabilities } from 'wagmi/experimental';
+
+export function useIsAuxiliaryFundsEnabled(): boolean {
+  const { chainId } = useAccount();
+  const { data } = useCapabilities();
```

Comment:
Should prob use @kirkas 's `apps/web/src/hooks/useCapabilitiesSafe.ts` here

```suggestion
  const { data } = useCapabilitiesSafe();
```

Reply:
Looks like you'd need to add `auxiliaryFunds` to its check

Reply:
yes and feel free to modify useCapabilitiesSafe to give you a auxiliaryFunds boolean